DFW młodzi wilcy + Paterek Ryucon (#2748)
tpwres/teczki-polskiego-wrestlingu: content/e/low/2026-07-10-low-6.md

* The show's poster published on 26.05.2026 featured English wrestler Trent Seven, announcing his participation in the show. Seven was previously seen at [PTW #3](@/e/ptw/2022-11-26-ptw-3-legends.md), where he fought [Joe Hendry](@/w/joe-hendry.md) for the IMPACT Digital Media Championship.
* Following [Tony Sheen's](@/w/tony-sheen.md) loss to [Marcelito](@/w/marcelito.md) (after a distraction by [Disco Pablo](@/w/disco-pablo.md)), angry [Arek Paterek](@/w/arek-paterek.md) [promised revenge][arek-sie-wkurzyl] invoking a biblical quote from Ezekiel 25:17 (in fact quoting [Samuel L. Jackson's monologue][25-17] from _Pulp Fiction_). He challenged them both to a six-man tag team match against Biuro Prasowe. Confronted with the fact that Biuro Prasowe consists of only one wrestler, he presented his newest acquisition: [Eryk Lesak](@/w/eryk-lesak.md).
* With the math still not working out for a six-man match, Paterek toured Poland to recruit the third member of his team. On 3.06.2026 Biuro posted a [video][paskal-nie-prasuje] of his attempt to recruit [Markus Paskal](@/w/markus-paskal.md), which backfired.

> * On 14.06.2026 Paterek posted a video of him visiting a [DFW](@/o/dfw.md) training, after getting a tip-off about some "young guns" (_młode wilki_, lit. "young wolves") being available for recruitment. [PJ Blake](@/w/pj-blake.md), acting as the leader, briefly presented thee of his trainees, but Paterek did not see any of them as fit to represent Biuro Prasowe. (The term "young wolves" refers to a [Polish film of the same name][mlodzi-wilcy], whereas Blake's lines are a nod to a scene from [Kiler][kiler] parodying Young Wolves.)

Reviewer: three for thee

Reviewer 2: Thou art correct!

* At the same show, [Syriusz Dziedzic](@/w/dziedzic.md) challenged [Sinister](@/w/sinister.md) to a match at Ryucon under the condition that if he wins, Dziedzic will once again join him. Dziedzic countered with a stipulation of his own: if he wins, Sinister must quit Legacy forever.
* The show's poster published on 26.05.2026 featured English wrestler Trent Seven, announcing his participation in the show. Seven was previously seen at [PTW #3](@/e/ptw/2022-11-26-ptw-3-legends.md), where he fought [Joe Hendry](@/w/joe-hendry.md) for the IMPACT Digital Media Championship.
* Following [Tony Sheen's](@/w/riverman.md) loss to [Marcelito](@/w/marcelito.md) (after a distraction by [Disco Pablo](@/w/disco-pablo.md)), angry [Arek Paterek](@/w/arek-paterek.md) [promised revenge][arek-sie-wkurzyl] invoking a biblical quote from Ezekiel 25:17 (in fact quoting [Samuel L. Jackson's monologue][25-17] from _Pulp Fiction_). He challenged them both to a six-man tag team match against Biuro Prasowe. Confronted with the fact that Biuro Prasowe consists of only one wrestler, he presented his newest acquisition: [Eryk Lesak](@/w/eryk-lesak.md).

> * With the math still not working out for a six-man match, Paterek toured Poland to recruit the third member of his team. On 3.06.2026 Biuro posted a [video][paskal-nie-prasuje] of his attempt to recruit [Markus Paskal](@/w/markus-paskal.md), which backfired.

Reviewer: Riverman

> * On 14.06.2026 Paterek posted a video of him visiting a [DFW](@/o/dfw.md) training, after getting a tip-off about some "young guns" (_młode wilki_, lit. "young wolves") being available for recruitment. [PJ Blake](@/w/pj-blake.md), acting as the leader, briefly presented three of his trainees, but Paterek did not see any of them as fit to represent Biuro Prasowe. (The term "young wolves" refers to a [Polish film of the same name][mlodzi-wilcy], whereas Blake's lines are a nod to a scene from [Kiler][kiler] parodying Young Wolves.)

Reviewer: ```suggestion
* On 14.06.2026 Paterek posted a video of him visiting a nondescript industrial building, where he discovers a [DFW](@/o/dfw.md) training session, after getting a tip-off about some "young guns" (_młode wilki_, lit. "young wolves") available to recruit. [PJ Blake](@/w/pj-blake.md), acting as the leader, briefly presented three of his trainees, but Paterek did not see any of them as fit to represent Biuro Prasowe. (The term "young wolves" refers to a [Polish movie of the same name][mlodzi-wilcy], whereas Blake's lines are a nod to a scene from another movie, [Kiler][kiler] parodying Young Wolves.)
```

> * On 14.06.2026 Paterek posted a video of him visiting a nondescript industrial building, where he discovers a [DFW](@/o/dfw.md) training session, after getting a tip-off about some "young guns" (_młode wilki_, lit. "young wolves") available to recruit. [PJ Blake](@/w/pj-blake.md), acting as the leader, briefly presented three of his trainees, but Paterek did not see any of them as fit to represent Biuro Prasowe. (The term "young wolves" refers to a [Polish movie of the same name][mlodzi-wilcy], whereas Blake's lines are a nod to a scene from another movie, [Kiler][kiler] parodying Young Wolves.)

Reviewer: ```suggestion
* On 14.06.2026 Paterek posted a [video of him visiting a nondescript industrial building][paterek-w-garazu], where he discovers a [DFW](@/o/dfw.md) training session, after getting a tip-off about some "young guns" (_młode wilki_, lit. "young wolves") available to recruit. [PJ Blake](@/w/pj-blake.md), acting as the leader, briefly presented three of his trainees, but Paterek did not see any of them as fit to represent Biuro Prasowe. (The term "young wolves" refers to a [Polish movie of the same name][mlodzi-wilcy], whereas Blake's lines are a nod to a scene from another movie, [Kiler][kiler] parodying Young Wolves.)
```

* [Ryucon's website](https://ryucon.pl/legacy-of-wrestling/)

[tournee-lesaka]: https://www.facebook.com/ErykLesak/posts/pfbid032n6ggQ1JrhoZpTotwhyQuuwB5zEgGJ6Vh237Nw8LCMZg1DvSQ3gA4hYtWR82Xnfgl
[arek-sie-wkurzyl]: https://www.instagram.com/biuro.prasowe/reel/DZC_FKVouJj/

> [25-17]: https://genius.com/Samuel-l-jackson-ezekiel-25-17-dialogue-annotated

Reviewer: Offering a better link here: https://en.wikipedia.org/wiki/Bodyguard_Kiba_(1973_film)#In_popular_culture

> [25-17]: https://genius.com/Samuel-l-jackson-ezekiel-25-17-dialogue-annotated

Reviewer: ```suggestion
[25-17]: https://en.wikipedia.org/wiki/Bodyguard_Kiba_(1973_film)#In_popular_culture

[paskal-nie-prasuje]: https://www.instagram.com/p/DZHfkXOM-Tx/
[mlodzi-wilcy]: https://www.imdb.com/title/tt0113835/

> [kiler]: https://en.wikipedia.org/wiki/Kiler_(film)

Reviewer: ```suggestion
[kiler]: https://en.wikipedia.org/wiki/Kiler_(film)
[paterek-w-garazu]: https://www.instagram.com/p/DZkd2EWoJ6n/
```
